docs/source/savant_101/54_conditional_unit_init.rst

```diff
@@ -0,0 +1,24 @@
+Conditional Unit Initializing
+=============================
+
+Single module config can contain multiple variations of the pipeline through the use of conditionally initialized :py:class:`~savant.config.schema.ElementGroup`.
+
+For example, the user may introduce alternative detector units without the need to duplicate all the other module parameters in a separate config file, as is done in the `traffic meter <https://github.com/insight-platform/Savant/tree/develop/samples/traffic_meter>`__ sample. Here's a section of module config from the beginning of the pipeline definition that adds a pyfunc unit as usual, and then defines a conditionally enabled group for the peoplenet detector:
+
+.. literalinclude:: ../../../samples/traffic_meter/module.yml
+  :language: YAML
+  :lines: 27-45
+
+Later in the module config another group is defined for the yolov8m detector:
+
+.. literalinclude:: ../../../samples/traffic_meter/module.yml
+  :language: YAML
+  :lines: 67-75
+
+The groups are initialized based on whether their :py:attr:`~savant.config.schema.ElementGroup.init_condition` evaluates to `True` at the time of module initialization. In the example above which detector unit is initialized depends on the value of the ``DETECTOR`` environment variable. If it is set to `peoplenet` then the peoplenet detector is initialized, and similarly for the yolov8m detector.
```

Comment:
Describe what syntax is used to evaluate init_condition expression

Reply:
Added an explanation.